stable/jenkins/templates/_helpers.tpl

```diff
@@ -159,6 +129,69 @@ unclassified:
     url: {{ template "jenkins.url" . }}
 {{- end -}}
 
+{{/*
+Returns agent (podTemplate) configuration as code
+*/}}
+{{- define "jenkins.casc.agent" -}}
+{{- if .Values.agent.enabled -}}
+- name: "{{ .Values.agent.podName }}"
```

Comment:
If we use Jenkins inheritance for agents then we could simply include yaml below this agent configuration.

Reply:
Sorry. I don't understand what you're suggesting here.